docs/operate/customize/sitemaps.md

```diff
@@ -2,26 +2,27 @@
 
 _Introduced in v13_
 
-[Sitemaps](https://sitemaps.org/) are a longstanding way to layout the pages of a website in a format that a search engine (or other such crawler) can efficiently parse and, in turn, serve as search results. They make your instance's content directly available to search engines even if no other pages on the web have linked to them yet and can even enhance your standing in a search engine's offerings.
+[Sitemaps](https://sitemaps.org/) are a longstanding way to list the pages of a website in a format that a search engine (or other such crawler) can efficiently parse and, in turn, use to provide search results. They make your instance's content directly available to search engines even if no other pages on the web have linked to them yet and can even enhance your standing in a search engine's offerings.
 
-In InvenioRDM, sitemaps are automatically generated and updated. This covers both sitemap indices (groupings of sitemaps) and sitemaps themselves. These indices and the generation of multiple sitemaps are essential, because a single sitemap can only contain a limited number of pages (50,000) and be of a certain size (under 50MB). This is all taken care of automatically, which allows your site to be discoverable no matter its scale.
+In InvenioRDM, sitemaps are automatically generated and updated. This covers both sitemap indices (groupings of sitemaps) and sitemaps themselves. These indices and the generation of multiple sitemaps are essential, because a single sitemap can only contain a limited number of pages (50,000 maximum) and be of a certain size (50MB maximum). This is all taken care of automatically by InvenioRDM, which allows your site to be discoverable no matter its scale.
 
 ## Location
 
 The relevant sitemaps-related links for your instance are found (at the root of your instance's domain/URL prefix):
 
 - **Sitemap indices**: `/sitemap_index_<int:page>.xml`
 - **Sitemaps**: `/sitemap_<int:page>.xml`
+- **Root sitemap**: `/sitemap.xml`
```

Comment:
I guess we're setting `SITEMAP_ROOT_VIEW_ENABLED = True` in the base app-rdm config? Should I have actually done this in `invenio-sitemap` already?

Reply:
ha! I thought that had been set to True by default :facepalm: . I can revert that or make a note about the setting?

Reply:
I can release `invenio-sitemaps v0.3.0` with the default set to `True` (zero-based versioning means it's a breaking change, but I think it's fine). Someone was pretty smart to pin in `invenio-app-rdm`, `invenio-sitemap>=0.1.0,<2.0.0` 😏